src/preferences/dialog/dlgpreflibrarydlg.ui

```diff
@@ -249,6 +249,13 @@
         </property>
        </widget>
       </item>
+      <item row="4" column="0">
+       <widget class="QCheckBox" name="checkBoxEditMetadata">
+        <property name="text">
+         <string>Edit metadata</string>
```

Comment:
This string should have more context. How about "Edit metadata after clicking selected track"?

Reply:
For 2.1, I would like to keep this text, because we already have translations for it. 
It is not too bad and details can go to a Tooltip 
```
This enables instant metadata editing within the track table. Editing starts when clicking on an already selected item.
```

Reply:
The only recent added string is translated just for ``DE``so far. You can easily change it again.

Reply:
OK thank you.